docs/pages/v2/sources/utxorpc.mdx

```diff
@@ -0,0 +1,31 @@
+# UtxoRPC with Dolos
+
+The UtxoRPC with Dolos (UtxoRPC) source uses gRPC to fetch blocks and receive blocks from a no Dolos.
+
+## Configuration
+
+The following snippet shows an example of how to set up a typical UtxoRPC source:
+
+```toml
+[source]
+type = "UtxoRPC"
+url = ["https://<hostname>"]
+max_items = 20
```

Comment:
lets rename `max_items` into `max_items_per_page` to avoid confusion

Reply:
done